packages/spor-datepicker-react/src/DatePicker.tsx

```diff
@@ -0,0 +1,129 @@
+import {
+  Box,
+  FormLabel,
+  InputGroup,
+  Popover,
+  PopoverAnchor,
+  PopoverArrow,
+  PopoverBody,
+  PopoverContent,
+  ResponsiveValue,
+  useBreakpointValue,
+  useMultiStyleConfig,
+} from "@chakra-ui/react";
+import { DateValue } from "@internationalized/date";
+import { useDatePicker } from "@react-aria/datepicker";
+import { useDatePickerState } from "@react-stately/datepicker";
+import { CalendarOutline24Icon } from "@vygruppen/spor-icon-react";
+import React, { useRef } from "react";
+import { AriaDatePickerProps } from "react-aria";
+import { Calendar } from "./Calendar";
+import { CalendarTriggerButton } from "./CalendarTriggerButton";
+import { DateField, StyledField } from "./DateField";
+
+type DatePickerProps = AriaDatePickerProps<DateValue> & {
+  variant: ResponsiveValue<"simple" | "with-trigger">;
+};
+/**
+ * A date picker component.
+ *
+ * There are two versions of this component – a simple one, and one with a trigger button for showing the calendar. Use whatever fits your design.
```

Comment:
`with-trigger` er ment for bruk hvor man har on screen keyboard osv... Er det verd å nevne i docs? Ikke nødvendigvis her, men et eller annet sted?

Reply:
Pratet litt med Per Øyvind – det var ikke alltid så svart hvitt heller :/ Enig i at det burde nevnes (gjør forsåvidt det på et vis allerede, men kan sikkert utbrodere mer).